### DISCUSSION TITLE
Loading smaller model

### DISCUSSION THREAD

**Discussion by TobiGee** [snakers4/silero-vad#191] [Q&A]
Hi guys,
I have loaded silero VAD by doing
`torch.hub.load(repo_or_dir='snakers4/silero-vad', model='silero_vad', force_reload=False)` 
After analyzing the consumption of resources i found that i want to test smaller Silero models.
The wiki tells me there is a mini (v2.3) and a micro release (v2.2).
Am not very experienced with pytorch and think there is some trick to obtain all the other available models.
Can anyone tell me how to do that!
Thanks very much in advance! :)

**Comment by snakers4** [comment]
Hi,

The new V3 [model](https://github.com/snakers4/silero-vad/issues/2#issuecomment-987878454) superseded all of the previous models.

All old small models are obsolete.

The new model is "medium", but if I remember correctly it is only 100k params, which is small for ML models.

**Reply by snakers4** [reply]
@TobiGee
